report.py

```diff
@@ -107,6 +109,8 @@
 OPERATOR_XDAI_BALANCE_THRESHOLD = 50000000000000000
 MECH_REQUESTS_PER_EPOCH_THRESHOLD = 10
 TRADES_LOOKBACK_DAYS = 3
+REBETS_LOOKBACK_DAYS = 3
+DAY_IN_UNIX = 60 * 60 * 24
```

Comment:
```suggestion
SECONDS_PER_DAY = 60 * 60 * 24
```

Reply:
Why?

Reply:
because this is what the constant is about.